SensorMap.cpp

```diff
@@ -38,42 +38,197 @@ double SensorMap::getDistanceLeftDown() {
   return sonar[5].ping_cm(); 
 }
 
+double SensorMap::getPwmUltrasonicLeft() { // Positive.
```

Comment:
Any function regarding actually computing the error should probably be in Control.h and Control.cpp